subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidStudioMockupPerformanceTest.groovy

```diff
@@ -26,11 +26,10 @@ class RealLifeAndroidStudioMockupPerformanceTest extends AbstractAndroidStudioMo
         given:
 
         experiment(testProject) {
-            minimumVersion = "3.4"
-            targetVersions = ["4.2-20170817235727+0000"]
+            minimumVersion = "4.3.1"
+            targetVersions = ["4.3.1"]
             action('org.gradle.performance.android.SyncAction') {
                 jvmArguments = customizeJvmOptions(["-Xms2g", "-Xmx2g"])
-                withArguments("android.builder.sdkDownload=true", "-Dcom.android.build.gradle.overrideVersionCheck=true")
```

Comment:
I pushed all the arguments up into the SyncAction. `withArguments´ overwrites the arguments.